governance/polls/Prioritization Sentiment - Technical Resource Deployment - June 6, 2022.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Prioritization Sentiment - Technical Resource Deployment - June 6, 2022
+summary: Rank your Techical Resourse Priority Sentiment to indicate where you believe MakerDAO should be focusing resources. 
+discussion_link: https://forum.makerdao.com/t/prioritization-framework-sentiment-polling/15554
+vote_type: Ranked Choice IRV
+categories:
+   - Misc Governance
+   - Priority Sentiment
+options:
+   0: Abstain
+   1: Flapper Distributor
+   2: Sagittarius Engine
+   3: Collateral Onboarding
+   4: Compound D3M
+   5: Term Lending Module (TLM)
+   6: Maker Teleport
+   7: DssKiln
+   8: DssSnog
+   9: DssGate
+   10: Target Price Adjustment Module (Vox)
+   11: ESM Simulations
+   12: Returning "Lost" DAI
+start_date: 2022-06-06T16:00:00
+end_date: 2022-07-04T16:00:00
+---
+# Poll: Prioritization Sentiment - Technical Resource Deployment - June 6, 2022
+
+The Governance Facilitators have placed a Priority Sentiment Poll into the voting system on behalf of MKR holders. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for 4 weeks beginning on Monday, June 6 at 16:00 UTC.
+
+
+**This is an instant-runoff vote.** 
+- **You may rank any number of options that you would support for priority consideration.** 
+- **Unranked options signal that you would not support the option(s) for priority consideration.**
+- **If you have no priority preference to any of the listed options, you should vote 'Abstain' as your first and only choice.**
+
+## Review
+
+The community can vote in this poll to rank their support of the following options:
+- [Flapper Distributor](https://forum.makerdao.com/t/flapperdistributor-a-way-to-distribute-system-surplus-while-minimizing-governance/4591) - A way to split revenue and direct it toward different causes, like buyback/burn and filling the surplus buffer simultaneously. 
+- [Sagittarius Engine](https://forum.makerdao.com/t/sagittarius-engine/10852) - Tokenomics design from Rune, including staking MKR and the ability for those with MKR staked to borrow against it.
```

Comment:
So things like Sagittarius Engine have never been voted on, I guess we need to include somewhere that this would entail someone crafting a proposal to go to a vote if prioritized rather than technical work

Reply:
Added a line to this and Flapper, worded so it wouldnt apply to the Dss Modules. We'll see what people think :)